app/modules/jwplayer-assets.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+
+const {RSVP} = Ember;
+
+export const assetUrls = {
+	styles: '/mobile-wiki/assets/jwplayer/index.css',
+	script: '/mobile-wiki/assets/jwplayer/wikiajwplayer.js'
```

Comment:
Remember to test on a sandbox if these are fingerprinted correctly. There were issues in the past.

Reply:
👍